### ISSUE TITLE
Require 4.0.3 for latest Bioconductor 3.12  

### ISSUE THREAD

**Issue by vondoRishi** [CSCfi/rstudio-openshift#2]
Hi,

I like to use Bioconductor 3.12  which requires R 4.0.3. I have tried to use "rocker/tidyverse:4.0.**3**" in the [Dockerfile](https://github.com/CSCfi/rstudio-openshift/blob/52c1359f66eb17150ec74760f4d018993b5e5b07/server/Dockerfile#L2). However, it has caused an error similar to this [issue](https://github.com/rocker-org/rocker-versioned2/issues/105).

Now I am using "rocker/tidyverse:4.0.**2**" but it would be nice to have a solution for R 4.0.3 

Thanks

**Comment by koetjen** [comment]
Hi,

I've rebuilt the rocker installation for R 4.0.3 to support Bioconductor 3.12 on our cluster by resetting to RStudio 1.3. It's available to pull at koetjen/rstudio:4.0.3 

Dockerfile:
```

FROM rocker/r-ver:4.0.3-cuda10.1
LABEL org.label-schema.license="GPL-2.0" \
      org.label-schema.vcs-url="https://github.com/rocker-org/rocker-versioned" \
      org.label-schema.vendor="Rocker Project" \
      maintainer="Carl Boettiger <cboettig@ropensci.org>"
ENV S6_VERSION=v2.0.0.1
ENV RSTUDIO_VERSION=1.3.1093
ENV PANDOC_VERSION=default
ENV TENSORFLOW_VERSION=gpu
ENV KERAS_VERSION=default
ENV PATH=/usr/lib/rstudio-server/bin:$PATH
RUN /rocker_scripts/install_rstudio.sh
RUN /rocker_scripts/install_pandoc.sh
RUN /rocker_scripts/install_tidyverse.sh
RUN /rocker_scripts/install_tensorflow.sh
RUN /rocker_scripts/install_verse.sh
RUN /rocker_scripts/install_geospatial.sh
EXPOSE 8787
CMD ["/init"]

```

**Comment by vondoRishi** [comment]
Thanks a lot. Additionally, I have added few more lines to add shiny-server.
Please let me know if you have any comments regarding this.

> FROM rocker/r-ver:4.0.3-cuda10.1
LABEL org.label-schema.license="GPL-2.0" \
      org.label-schema.vcs-url="https://github.com/rocker-org/rocker-versioned" \
      org.label-schema.vendor="Rocker Project" \
      maintainer="Carl Boettiger <cboettig@ropensci.org>"
ENV S6_VERSION=v2.0.0.1
ENV RSTUDIO_VERSION=1.3.1093
ENV PANDOC_VERSION=default
ENV TENSORFLOW_VERSION=gpu
ENV KERAS_VERSION=default
`ENV SHINY_SERVER_VERSION=latest`
ENV PATH=/usr/lib/rstudio-server/bin:$PATH
RUN /rocker_scripts/install_rstudio.sh
RUN /rocker_scripts/install_pandoc.sh
RUN /rocker_scripts/install_tidyverse.sh
RUN /rocker_scripts/install_tensorflow.sh
RUN /rocker_scripts/install_verse.sh
RUN /rocker_scripts/install_geospatial.sh
`RUN /rocker_scripts/install_shiny_server.sh`
EXPOSE `3838` 8787
CMD ["/init"]
